bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Accordion.java

```diff
@@ -50,7 +50,7 @@ default boolean isSingleExpansion() {
      * @since com.adobe.cq.wcm.core.components.models 12.8.0
      */
     default String[] getExpandedItems() {
-        throw new UnsupportedOperationException();
+        return null;
```

Comment:
Shouldn't all the default methods that now return `null` instead of an exception have a `@Nullable` Annotation to mark the method as potentially returning a `null` value ?

Reply:
I believe most static analysis tools will treat no annotation as `@Nullable` for method return types. I see no point in being explicit here. The contract is more important for `@NotNull`.

Reply:
ok